main_sweeps.py

```diff
@@ -0,0 +1,173 @@
+import argparse
+import json
+import os
+import shutil
+from typing import Any, Dict, Type
+
+import numpy as np
+import pandas as pd
+from pyntcloud import PyntCloud
+from pyquaternion import Quaternion
+
+from argoverse.utils.se3 import SE3
+from argoverse.utils.transform import quat2rotmat
+from main import get_calibration_info, round_to_micros, extract_pc
+from nuscenes.nuscenes import NuScenes
+
+"""
+Converts the nuScenes dataset into the Argoverse format.
+
+In the nuScenes dataset, samples contain annotated data sampled at a frequency
+of 2Hz, whereas sweeps contains all the unannotated data. The capture frequency
+for camera is 12Hz and for lidar it 20Hz. Lidar data is provided in lidar sensor
```

Comment:
small typo here -- "for LiDAR it is 20 hz"

Reply:
Done